src/client.js

```diff
@@ -94,6 +94,27 @@ function Client(manager, name, config) {
 
 	if (typeof client.config.sessions !== "object") {
 		client.config.sessions = {};
+	} else {
+		// TODO: This is just for backwards compatibility. Remove in v3.0.0
```

Comment:
If I'm reading this correctly, this means that once we delete it, anyone upgrading from 2.6 to whatever version doesn't have it in (3.0.0 currently), their auth won't work?

Reply:
They will simply have to re-login. We've already had this happen before (2.5 I think?)

My suggestion is to tell people to upgrade to `2.x` before going to `3.x` for existing users because we have quite a bit of deprecation warnings.

Reply:
I'll make sure to mention this in changelog(s). And yeah, it only means people will have to re-login, so that's a reasonable worst case solution :)

Reply:
Ah right, yes, of course, this is the session tokens, not the user password. Minor mind-slip there.